index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Website</title>
+</head>
+<body >
+    <header class="header-logo">
+        <h2 class="logo">My Logo</h2>
+        <img src="images/menu-rounded.png" alt="" class="image-icon"> 
+         
+    </header>
+
+    <div class="all-body">
+        <section class="main">
+            <h2 class="title">I'm Mariane Glad to see you!</h2>
+            <p class="intro">I'm a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't 
+                hestiate to contact me.</p>
+        </section>
+
+        <div class="connect">
+
+            <div class="lets-connect">
+                <p>LET'S CONNECT</p>
+            </div>
+
+            <img src="images/Vector.png" class="icons">
+            <img src="images/Vector (1).png" class="icons"><img src="images/Vector (2).png" class="icons">
+            <img src="images/Vector (3).png" class="icons">
+            <img src="images/Vector (4).png" class="icons">
+            <img src="images/Vector (5).png" class="icons">
+        </div>
+    </div>
```

Comment:
- [ ] Dear @MarkYegon7, I notice you wrap part of your heading into a section tag, Please notice everything under the heading is a single section and should be treated as that. Please wrap all your section in a `section` tag instead of `div`.
- [ ] Please also don't forget to use the right font family and size for your texts, especially your heading text, as you can see your design is not fit into a single view on a `375px`  screen size.
- [ ] Aside from that, Your background image is not aligned to take the whole width of the display, If you notice on the figma design, you will see the background take all the space from left to right. Please make sure to align your background properly. 

### Figma design 
![image](https://user-images.githubusercontent.com/69077061/219777139-1c0f1e27-9155-4885-b40b-93aae85e8375.png)

### Your design.
![image](https://user-images.githubusercontent.com/69077061/219776960-7edb1429-2944-4ee2-948e-3a053ee1aa0a.png)